client/app/components/groups/group-name.js

```diff
@@ -23,7 +23,7 @@ export default function init(ngModule) {
     transclude: true,
     template: `
       <h2 class="m-t-0">
-        <edit-in-place editable="$ctrl.canEdit()" done="$ctrl.saveName" ignore-blanks='true' value="$ctrl.group.name"></edit-in-place>&nbsp;
+        <edit-in-place class="edit-in-place" editable="$ctrl.canEdit()" onDone="$ctrl.saveName" ignore-blanks="'true'" value="$ctrl.group.name"></edit-in-place>&nbsp;
```

Comment:
Why do we need to specify the class name?

`onDone` should probably be `on-done`, right?